File: java_i/tp3/src/Exercise04.java
import market.Product;

public class Exercise04 implements IExercise {
  @Override
  public String getName() {
    return "Testando a Classe “Produto”";
  }

  @Override
  public void run() {
    var pizza = new Product("Pizza Pepperoni", 49.90, 8);

    pizza.displayInfo();

    pizza.setPrice(47.99);
    pizza.setQuantityInStock(6);

    pizza.displayInfo();
  }
}

File: c_sharp_i/TP3/Exercise04.cs
namespace TP3;

public class Exercise04 : IExercise
{
  public string Name => "Testando a Classe \"Ingresso\"";

  public void Run(string[] args)
  {
    var ticket = new Ticket
    {
      Price = 20f,
      Name = "Skillet",
      Quantity = 200
    };

    ticket.DisplayInfo();

    ticket.ChangePrice(39.99);
    ticket.ChangeQuantity(42);

    ticket.DisplayInfo();
  }
}
